packages/web/src/common/store/pages/track/sagas.js

```diff
@@ -171,7 +170,11 @@ function* watchFetchTrack() {
         // Add hero track to lineup early so that we can play it ASAP
         // (instead of waiting for the entire lineup to load)
         yield call(addTrackToLineup, track)
-        yield fork(getRestOfLineup, permalink, handle)
+        yield fork(
+          getRestOfLineup,
+          track.permalink,
```

Comment:
All tracks have a `permalink` field right? cc @raymondjacobson

Reply:
@rickyrombo might have more context here too

Reply:
@rickyrombo do u kno ^ :>)?

Reply:
Yes they should!